apps/evalite-ui/app/routes/$.tsx

```diff
@@ -0,0 +1,26 @@
+import { getMenuItems } from "@evalite/core/sdk";
+import { createFileRoute, redirect } from "@tanstack/react-router";
+
+export const Route = createFileRoute("/$")({
```

Comment:
This is a [catch-all route](https://tanstack.com/router/latest/docs/framework/react/guide/code-based-routing#splat-catch-all-routes), mostly unchanged from the original `_index` file and does the same thing (a redirect)